de-lib/src/main/java/org/iplantc/de/client/services/converters/AppTemplateDECallbackConverter.java

```diff
@@ -0,0 +1,165 @@
+package org.iplantc.de.client.services.converters;
+
+import org.iplantc.de.client.models.apps.integration.AppTemplate;
+import org.iplantc.de.client.models.apps.integration.AppTemplateAutoBeanFactory;
+import org.iplantc.de.client.models.apps.integration.Argument;
+import org.iplantc.de.client.models.apps.integration.ArgumentGroup;
+import org.iplantc.de.client.models.apps.integration.ArgumentType;
+import org.iplantc.de.client.models.apps.integration.ArgumentValidator;
+import org.iplantc.de.client.models.apps.integration.SelectionItem;
+import org.iplantc.de.client.models.apps.integration.SelectionItemGroup;
+import org.iplantc.de.client.util.AppTemplateUtils;
+import org.iplantc.de.shared.DECallback;
+
+import com.google.common.collect.Lists;
+import com.google.web.bindery.autobean.shared.AutoBean;
+import com.google.web.bindery.autobean.shared.AutoBeanCodex;
+import com.google.web.bindery.autobean.shared.AutoBeanUtils;
+import com.google.web.bindery.autobean.shared.Splittable;
+import com.google.web.bindery.autobean.shared.impl.StringQuoter;
+
+import java.util.List;
+
+/**
+ * @author jstroot
+ */
+public class AppTemplateDECallbackConverter extends DECallbackConverter<String, AppTemplate> {
```

Comment:
Yes, you're right.  Now that I'm looking at it, I think I might have missed a few spots, but the reason why I created a copy was not all the calls were going to a `/apps/*` endpoint and I was meaning to ask about those.  One of the calls goes to `/preview-template` for example and I wasn't sure if that was still going to the apps service.  If not, then I don't think it makes sense to disable the entire App window's functionality for that.